util/api.py

```diff
@@ -120,14 +122,19 @@ def error_internal(debug_info=None):
         def bad_request(debug_info=None):
             return Error('badrequest', 'An internal error occurred', debug_info=debug_info)
 
+        # Input is base64 encoded and compressed to reduce size (max rule length in iRODS is 20KB)
         # Validate input string: is it a valid JSON object?
         try:
-            data = jsonutil.parse(inp)
+            base64_decoded = base64.b64decode(inp)
```

Comment:
This could raise a binascii.Error: Invalid base64-encoded string: number of data characters (1) cannot be 1 more than a multiple of 4 when e.g. someone sends "a=" as inp?